activesupport/lib/active_support/cache.rb

```diff
@@ -477,6 +477,18 @@ def delete(name, options = nil)
         end
       end
 
+      # Deletes multiple entries in the cache. Returns the number of entries deleted.
```

Comment:
Should we introduce this requirement? Redis naturally returns the number of entries deleted, but it may be costly or impractical for other cache stores.

Reply:
The regular `delete_entry` returns a Boolean on whether it successfully deleted an entry or not, so I don't see why this would be more costly since for most cache stores it just calls `delete_entry` `n` times. Is there a particular cache store you think could get a performance bump if this requirement was dropped?

Reply:
We are introducing a new API. Do we care about its return value, or should we leave it undefined?

For example, consider a third-party cache store that supports batch deletes but doesn't report the number of actual deletions. Or a cache store that issues deletions asynchronously. Should these implementations attempt to return a count of deleted entries?

Reply:
Thanks for the explanation! I've updated it to keep the current logic but removed the return value from the documentation.